centreon/tests/api/features/UserFilter.feature

```diff
@@ -10,6 +10,9 @@ Feature:
   Scenario: User filters
     Given I am logged in
 
+    When I send a GET request to '/api/v21.10/users/filters/unknown-page'
+    Then the response code should be "500"
```

Comment:
Why not a 404 instead of 500 ?

Reply:
To me it doesn't qualify for a 404 error. The name 'pageName' is not very representative, I do not search an corresponding element in DB or anything. If I don't "search" I can't "found".

Reply:
If the endpoint 'users/filters/unknown-page' is what you're looking for, a 404 error is preferable to a 500 error.